xbuild-target/src/main.rs

```diff
@@ -0,0 +1,6 @@
+fn main() {
+    winit::event_loop::EventLoop::new()
+        .unwrap()
+        .run(|_, _| todo!())
```

Comment:
A real implementation is probably provided by the example that was being compiled before? I'd rather test that, _including_ the part that forwards `android_activity::AndroidApp` into `winit`.

Reply:
I added an actual Android winit app, but it looks like there is a linker error that doesn't occur on my machine.